next/src/types/_pages/PrivacyPolicyPage.ts

```diff
@@ -0,0 +1,12 @@
+import { type Node } from "../_global/Node";
+import { type Seo } from "../_global/Seo";
+
+export interface PrivacyPolicyPage {
+  _type?: string;
```

Comment:
`_type` shouldn't be required?

Reply:
I'm using the same type (PivacyPolicyPage) inside a component that does not use _type. I don't have to redeclare a very similar type to PrivacyPolicyPage inside that component. If it was required the _type had to be used inside the child component or I would have gotten eslint and typescript errors.